ENIGMAsystem/SHELL/Graphics_Systems/OpenGL-Common/surface.cpp

```diff
@@ -16,6 +16,8 @@
 *** with this code. If not, see <http://www.gnu.org/licenses/>
 **/
 
+#ifdef GL_DRAW_FRAMEBUFFER
```

Comment:
No, you must have messed this up, we had two versions of the FBO that works. GL1 just had `GL_DRAW_FRAMEBUFFER` but GL3 has a separate `GL_DRAW_FRAMEBUFFER` and `GL_READ_FRAMEBUFFER`. At least the former should work in GLES2.

Reply:
I see where the confusion is. For GLES 2 it's just `GL_FRAMEBUFFER` like it is for GL1. For later it's both `GL_DRAW_FRAME_BUFFER` and `GL_READ_FRAMEBUFFER`.
https://www.khronos.org/registry/OpenGL-Refpages/es2.0/xhtml/glBindFramebuffer.xml

Reply:
I can try adding a define GL_DRAW_FRAMEBUFFER GL_FRAMEBUFFER later I guess

Reply:
seems to compile w/ defines but no idea if it works